src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectModulesController.kt

```diff
@@ -0,0 +1,106 @@
+package com.terraformation.backend.accelerator.api
+
+import com.terraformation.backend.accelerator.db.ModuleStore
+import com.terraformation.backend.accelerator.model.EventModel
+import com.terraformation.backend.accelerator.model.ModuleModel
+import com.terraformation.backend.api.AcceleratorEndpoint
+import com.terraformation.backend.api.ApiResponse200
+import com.terraformation.backend.api.ApiResponse404
+import com.terraformation.backend.db.accelerator.CohortPhase
+import com.terraformation.backend.db.accelerator.EventId
+import com.terraformation.backend.db.accelerator.EventType
+import com.terraformation.backend.db.accelerator.ModuleId
+import com.terraformation.backend.db.default_schema.ProjectId
+import io.swagger.v3.oas.annotations.Operation
+import java.net.URI
+import java.time.Instant
+import java.time.LocalDate
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@AcceleratorEndpoint
+@RequestMapping("/api/v1/modules")
+@RestController
+class ProjectModulesController(
+    private val moduleStore: ModuleStore,
+) {
+  @ApiResponse200
+  @ApiResponse404
+  @GetMapping("/projects/{projectId}")
+  @Operation(summary = "Gets modules for a project.")
+  fun listModules(
+      @PathVariable("projectId") projectId: ProjectId,
+  ): GetProjectModulesResponsePayload {
+    val models = moduleStore.fetchModulesForProject(projectId)
+    return GetProjectModulesResponsePayload(models)
+  }
+}
+
+data class ProjectModuleEventSession(
+    val id: EventId,
+    // Times
```

Comment:
Are the comments like this one just to split the fields up into sections visually? I'm not sure they're adding any information; hopefully it's self-evident that `startTime` and `endTime` are times.

If we want to document what the fields are, we could add `@Schema` annotations so the docs appear in the OpenAPI schema rather than just the source code. Though again, I think most of the field names in these payloads are pretty self-explanatory.